jenkins/resource_jenkins_job.go

```diff
@@ -124,3 +128,14 @@ func resourceJenkinsJobDelete(ctx context.Context, d *schema.ResourceData, meta
 	log.Printf("[DEBUG] jenkins::delete - %q removed: %t", name, ok)
 	return nil
 }
+
+func resourceJenkinsJobImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+        name := formatJobName(d.Id())
```

Comment:
Hmm, not sure that fixed it @hgibsonqb? Now there are extra `job/` lines in the final item.

Here are exact steps to recreate, using the Dockerized Jenkins in the `example` folder and assuming Terraform 0.13:
```sh
# Checkout branch
git checkout feature/make-job-importable

# Build provider
go install

# Initialize example directory and copy binary in (assumes Terraform 0.13)
cd example/
terraform init
ln -sf $(go env GOPATH)/bin/terraform-provider-jenkins ./.terraform/plugins/registry.terraform.io/taiidani/jenkins/0.6.0/$(go env GOOS)_$(go env GOARCH)/terraform-provider-jenkins_v0.6.0

# Spin up Jenkins and associated resources
docker-compose up -d
terraform apply --auto-approve
open http://localhost:8080 || echo "Could not open URL. You will have to browse to http://localhost:8080 manually"

# Remove the "freestyle" job from the state and re-import it
terraform state rm jenkins_job.freestyle
terraform import jenkins_job.freestyle folder-name/freestyle || 'echo Non "/job/" format failure expected'
terraform import jenkins_job.freestyle folder-name/job/freestyle

# Now run a plan and look at the output
terraform plan
```

The output of my plan is saying:
```sh
 # jenkins_job.freestyle must be replaced
-/+ resource "jenkins_job" "freestyle" {
      ~ id         = "folder-name/job/freestyle" -> (known after apply)
      ~ name       = "folder-name/job/job/job/freestyle" -> "folder-name/freestyle" # forces replacement
```

Note the extra "job/job/job" in the "name" property.

```suggestion
        name := strings.ReplaceAll(d.Id(), "/job/", "/")
```